libpod/runtime_pod_linux.go

```diff
@@ -178,11 +178,19 @@ func (r *Runtime) removePod(ctx context.Context, p *Pod, removeCtrs, force bool)
 		return err
 	}
 
-	numCtrs := len(ctrs)
+	var newCtrs []*Container
```

Comment:
I think we can do this in the loop below, after the check of len(ctrs) - we are getting into the loop because removeCtrs should be true

Reply:
I think that `removeCtrs` is set to true by default, when called by `podman rm`.